docs/validator.go

```diff
@@ -0,0 +1,167 @@
+// +build ignore
```

Comment:
I think we should build on `make` to ensure its compiling all the time.

Reply:
There is no `make` in minio-go, We ensure by running on travis.

Reply:
Added a make in minio-go